kornia/augmentation/augmentation.py

```diff
@@ -35,8 +32,6 @@
     adjust_contrast,
     adjust_saturation,
     adjust_hue,
-    adjust_gamma,
-    Invert,
```

Comment:
you shouldn't remove those

Reply:
ok, just rracting to my IDE saying were not used